cypress-tests-v2/cypress/fixtures/business_profile.json

```diff
@@ -27,7 +27,8 @@
       "payment_created_enabled": true,
       "payment_succeeded_enabled": true,
       "payment_failed_enabled": true
-    }
+    },
+    "is_click_to_pay_enabled": false
```

Comment:
Did we necessarily have to pass this field, or does the application take this field to be `false` if we don't pass it?

Reply:
If we don't pass it, it is not taking this field as `false`. It shows that the field is missing and the message goes like the following
Business profile create call failed with status 400 and message: "Json deserialize error: missing field `is_click_to_pay_enabled`

I am attaching a screenshot I took earlier
![image](https://github.com/user-attachments/assets/8d8f13f8-4ec6-44dc-a81e-5b9effc660dd)

Reply:
@sahkal can you please fix this?

Reply:
@Narayanbhat166 

https://github.com/juspay/hyperswitch/pull/6935